tests/integration/favorites.rs

```diff
@@ -7,7 +7,7 @@ use assert_cmd::prelude::*;
 use indoc::indoc;
 use std::path::PathBuf;
 
-fn create_template(name: &str) -> Project {
+fn create_template(description: &str) -> Project {
```

Comment:
suggestion: I found it more reasonable to name those parameters along their meaning or name within the template, here we have the `project-name` not the `description`, I suggest to stick with the naming

Reply:
The argument is actually the description!

`project-name` is not being substituted here, but left for the template engine. `description` however is. Please notice this is inside a `format!` macro, thus the `{{project-name}} is ignored, while the `{}` is not.

Reply:
Thanks for clarifying